dist/stat_data.py

```diff
@@ -812,11 +812,13 @@ def __init__(self, name, desc, flags=''):
     CursorStat('cursor_next_hs_tombstone', 'cursor next calls that skip due to a globally visible history store tombstone'),
     CursorStat('cursor_next_skip_ge_100', 'cursor next calls that skip greater than or equal to 100 entries'),
     CursorStat('cursor_next_skip_lt_100', 'cursor next calls that skip less than 100 entries'),
+    CursorStat('cursor_next_skip_pages', 'cursor next call skipped pages without reading'),
```

Comment:
Maybe we can be more clear on the description. It is a count of pages that next call(s) are skipping without reading. It is not a count of next(s). Maybe reword as:

"Total number of pages skipped without reading by cursor next calls"

Similarly for the prev calls.

Reply:
I'd suggest renaming the statistic to `cursor_next_skip_page_count` or `cursor_next_skip_page_total` for clarity, and agree with Sulabh's suggestion to update the description.

Reply:
Fixed in a285b69.